### ISSUE TITLE
FR: Melbourne Based Coordination Server

### ISSUE THREAD

**Issue by zorflieg** [tailscale/tailscale#8028]
### What are you trying to do?

It would be amazing if Australia could get a Melbourne based server as well as Sydney.

### How should we solve this?

Setup up a Melbourne based server.

### What is the impact of not solving this?

Latency.

### Anything else?

Perth (West Coast) Server would also be advised.

**Comment by DentonGentry** [comment]
I assume this means a relay. https://tailscale.com/kb/1232/derp-servers/

There is only one coordination server, but there are relays scattered around the planet.

**Comment by zorflieg** [comment]
Yes sorry.Melbourne is Australia's 2nd most populous city and is on track to outgrow Sydney the most we'll know City.https://www.theage.com.au/national/victoria/melbourne-to-overtake-sydney-as-australias-largest-city-by-2031-20230104-p5cab6.htmlInternational companies often drop a PoP in Sydney and think 'job done'.But I advise as a 25yr engineering vet, a service in, in this case Relay, in Melbourne and (West Coast) Perth are always wise to encourage adoption.I know we are a small market but we are a technology savvy one.On May 3, 2023 1:27 PM, Denton Gentry ***@***.***> wrote:
I assume this means a relay. https://tailscale.com/kb/1232/derp-servers/
There is only one coordination server, but there are relays scattered around the planet.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
Message  protected by MailGuard: e-mail anti-virus, anti-spam and content filtering.https://www.mailguard.com.au

**Comment by bradfitz** [comment]
We try to keep DERP latency low for everybody, regardless of NAT traversal abilities. But our preferred hosting providers don't always offer POPs in all the places we'd like. We'll keep this in mind, but it might be a bit until we find a suitable host.

**Comment by zorflieg** [comment]
Yes very valid point.  For the ones I'm thinking about it seems to be these 'wework' style environments.After the pandemic many businesses here downsized and moved into these shared office environments.  They don't have control over how the wifi and network is setup.They have their sub-office pod thing with their little server in it.  They get 300mbps or so from their office using tailscale addresses as their sole destination, then they go to the shared boardroom on another floor or somewhere across the building and due to some networking choice of the building managers they are suddenly bouncing via another city and can't get a direct connection on 20mbps.I know adding a closer relay for these type of people is treating the symptom and not the cause but for SME's that are trying to leverage a scattered workforce tailscale is a great solution to simplify the complexity of these non traditional environments.  Where traditionally they would have had to have one destination address for one area and a whole second VPN style one for sometimes even just the next room because the building managers have say, vlanned every floor and added policy based routing in between.These would be paying tailscale customers.  Maybe really what I want is a feature that, even if it was just paying customers, they could auto detect and switch to a headscale relay instance based on closest latency vs the closest official relay.  From what I understand right now it's an 'either/or' and not an auto detect 'as well as', choice for private relays?On May 4, 2023 1:31 AM, Dito Windyaksa ***@***.***> wrote:
I think the Tailscale team would be more interested on why your devices aren't able to establish a direct connection ;-)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
Message  protected by MailGuard: e-mail anti-virus, anti-spam and content filtering.https://www.mailguard.com.au

**Comment by zorflieg** [comment]
I'm thankful just to have it considered.On May 4, 2023 4:40 AM, Brad Fitzpatrick ***@***.***> wrote:
We try to keep DERP latency low for everybody, regardless of NAT traversal abilities. But our preferred hosting providers don't always offer POPs in all the places we'd like. We'll keep this in mind, but it might be a bit until we find a suitable host.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
Message  protected by MailGuard: e-mail anti-virus, anti-spam and content filtering.https://www.mailguard.com.au

**Comment by wizzard0** [comment]
@zorflieg FYI you can host a custom DERP server for your tailnet(s) yourselves - eg I run one in Kyiv
